baus/summaries.py

```diff
@@ -141,6 +141,13 @@ def policy_activated(policy_loc, policy_nm, scenario):
                 "alternate_buildings_filter" in policy_loc else \
                 policy_loc["alternate_adjustment_formula"]
             write(policy_nm+" is activated with formula: {}".format(geog))
+        if scenario in policy_loc["enable_in_scenarios"] \
```

Comment:
This code has gotten wild as we've developed BAUS so much haha. 
I think this may need to be an `elif` if we want the code to continue on...